core/services/fluxmonitor/flux_monitor.go

```diff
@@ -149,13 +149,28 @@ func (fm *concreteFluxMonitor) Start() error {
 	return err
 }
 
+func (fm *concreteFluxMonitor) Ready() error {
+	if fm.started {
+		return nil
+	}
+	return errors.New("not started")
+}
+
+func (fm *concreteFluxMonitor) Healthy() error {
+	if fm.started {
+		return nil
+	}
+	return errors.New("not started")
```

Comment:
minor, but might be worth creating a constant value in some common package?

Reply:
I considered it, but it can't be in the `health` package since it results in an import loop. Maybe `utils`?

Reply:
`utils` or `models` would be suitable.